oauth_dropins/appengine_config.py

```diff
@@ -90,6 +91,7 @@ def read(filename):
 FLICKR_APP_SECRET = read('flickr_app_secret')
 GOOGLE_CLIENT_ID = read('google_client_id')
 GOOGLE_CLIENT_SECRET = read('google_client_secret')
+INDIEAUTH_CLIENT_ID = read('indieauth_client_id')
```

Comment:
this generally isn't secret, right? it's often just the site's url, e.g. `https://brid.gy/`? should clients maybe just pass it as a param somewhere instead?